O/Oceananigans/Compat.toml

```diff
@@ -4,3 +4,6 @@ GPUifyLoops = "0.1-0.2"
 JLD = "0.5-0.9"
 NetCDF = "0.1-0.7"
 julia = "1.1-1.3"
+
+["0.5.3"]
+julia = "1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.

Reply:
You can retrigger now.

Reply:
Awesome, thank you!